commands/import.go

```diff
@@ -159,20 +157,21 @@ func setRolling(dbExists bool, dbIsRolling bool, dbCurrChunk int, dbTotalChunks
 		}
 	}
 
-	// already existing db is converted to rolling if it wasn't already
-	// or if the user specified that a new database is rolling
-	cfg.Rolling = dbExists || userIsRolling
+	cfg.Rolling = dbIsRolling || userIsRolling
```

Comment:
This was an odd behavior that would convert any existing database to a rolling database.
This meant that if you ran an import command without any rolling parameters against an existing database, the cfg.Rolling flag would be set. This created problems with the `--delete` flag. 

If we run `rita import --delete existing_database` we do not want to be in rolling mode.

Reply:
I've updated the code to prevent upgrades unless the user specifies one of the rolling flags.  Since the user must set one of the rolling flags, `userIsRolling` will be true solving the issue here.